horizons/util/python/decorators.py

```diff
@@ -109,7 +109,8 @@ def __call__(self, *args, **kwargs):
 
 globals().update(opmap)
 
-def _make_constants(f, builtin_only=False, stoplist=[], verbose=False):
+def _make_constants(f, builtin_only=False, stoplist=None, verbose=False):
+	stoplist = stoplist or []
```

Comment:
I'd really prefer the check for `None`, as this this type of setting would use a new list instance instead of the given instance for an empty list. Even though that's probably not the best way to do it, one might want the original instance to be used (e.g. to see in tests, if items were added).